src/libraries/Microsoft.PowerFx.Core/Public/CheckResult.cs

```diff
@@ -588,8 +589,18 @@ public FormulaType GetNodeType(TexlNode node)
 
         // Called by language server to get custom tokens.
         // If binding is available, returns context sensitive tokens.  $$$
+        // Keeping this temporarily for custom publish tokens notification
+        // Feature to auto fix casing of function name in the editor also depends on this function and custom publish tokens notification
         internal IReadOnlyDictionary<string, TokenResultType> GetTokens(GetTokensFlags flags) => GetTokensUtils.GetTokens(this._binding, flags);
 
+        /// <summary>
+        /// Returns an enumeration of token text spans in a expression rule with their start and end indices and token type.
+        /// </summary>
+        /// <param name="comparer">Optional comparer to sort tokens.</param>
+        /// <param name="computeHiddenTokens">Optional flag to indicate whether to compute whether token can be hidden or not.</param>
+        /// <returns>Ordered or undordered collection of tokens.</returns>
+        internal ICollection<ITokenTextSpan> GetTokens(IComparer<ITokenTextSpan> comparer = null, bool computeHiddenTokens = false) => Tokenization.Tokenize(_expression, _binding, Parse.Comments, comparer, computeHiddenTokens);

```

Comment:
>ICollection [](http://example.com/codeflow?start=17&length=11)

why ICollection vs IEnumerable?

Reply:
I didn't pay attention to it really. I will change it to IEnumerable.

Reply:
Changed To IEnumerable